src/test/java/com/systelab/seed/user/UserServiceTest.java

```diff
@@ -13,46 +11,49 @@
 import com.systelab.seed.user.boundary.UserService;
 import com.systelab.seed.user.entity.User;
 import com.systelab.seed.user.entity.UserRole;
-import java.util.logging.Logger;
-import javax.inject.Inject;
 import org.jboss.arquillian.container.test.api.Deployment;
 import org.jboss.arquillian.junit.Arquillian;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import javax.inject.Inject;
+import java.util.logging.Logger;
+
+import static org.junit.Assert.assertNotNull;
+
 @RunWith(Arquillian.class)
 public class UserServiceTest {
 
-   @Inject
-   private UserService userService;
-
-   @Deployment
-   public static WebArchive createDeployment() {
-
-      // Create deploy file
-      return ShrinkWrap.create(WebArchive.class)
-          .addClasses(LoggerProducer.class, BaseException.class, BaseEntity.class, AuthenticationTokenGenerator.class, PasswordDigest.class, Pageable.class, Page.class,
-              Logger.class, RESTResourceTest.class)
-          .addAsResource("test-persistence.xml","META-INF/persistence.xml")
-          .addAsWebInfResource("jbossas-ds.xml")
-          .addAsResource("META-INF/beans.xml")
-          .addPackages(true, "com.systelab.seed.user", "org.apache.poi","com.systelab.seed.infrastructure.security", "com.systelab.seed.infrastructure.auth");
-   }
-
-   @Test
-   public void createUser() throws BaseException {
-      User user = new User();
-      user.setLogin("agoncalves");
-      user.setPassword("agoncalves");
-      user.setName("Antonio");
-      user.setSurname("Goncalves");
-      user.setRole(UserRole.ADMIN);
-
-      userService.create(user);
-
-      assertNotNull(userService.getUser(user.getId()));
-   }
+    @Inject
```

Comment:
Just a little comment: should we check the **formatter**? Because it seems that the number of spaces are different from the original one, thx!

Reply:
Waiting for new formater.

Reply:
We are working with @javintx in the new Code Formatter following the Google Code Rules Set. Once we have it, we will share it with the rest of people in the related repository.

By now, here is your approval! 👍